tests/integration/awslambda/test_lambda_common.snapshot.json

```diff
@@ -957,13 +957,13 @@
     }
   },
   "tests/integration/awslambda/test_lambda_common.py::TestLambdaRuntimesCommon::test_introspection_invoke[java8]": {
-    "recorded-date": "30-04-2023, 18:45:22",
+    "recorded-date": "02-05-2023, 17:50:12",
     "recorded-content": {
       "create_function_result": {
         "Architectures": [
           "x86_64"
         ],
-        "CodeSha256": "EIHcJLLHxO2t+7bCSyhzSs7ZoowNOLZ/OXtT8vw5Kr0=",
+        "CodeSha256": "eVn60S7b09aYSVdmCtubfLRgSaVdfZRpoVMWPWnbwCo=",
```

Comment:
I guess these are the OS-dependent zipping differences because we don't commit the ZIPs 🤔

Reply:
Exactly!